x-pack/plugin/esql/qa/testFixtures/src/main/resources/term-function.csv-spec

```diff
@@ -0,0 +1,190 @@
+###############################################
+# Tests for Term function
+#
+
+termWithTextField
+required_capability: term_function
+
+// tag::term-with-field[]
+from books 
+| where term(author, "gabriel")  
+| keep book_no, title
+| limit 3;
```

Comment:
```suggestion
FROM books 
| WHERE TERM(author, "gabriel")  
| KEEP book_no, title
| LIMIT 3;
```

Trivially, capitalise the commands which show up in the docs, so that they consistent with other examples.